### PR TITLE
Delete unused and fix a bit

### PR DESCRIPTION
**Pull request by hzlmn**

### CODE REVIEW

File: js/plugins.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-(function(global, doc, $) {
+(function(global) {
+  
+  var $ = global.jQuery || global.$;
```
Comment (arthurvr) [inline js/plugins.js]: Why the `|| global.$`?

**Comment by arthurvr** [comment]
Thanks for the PR!

File: js/plugins.js (hunk 1 above)
Comment (hzlmn) [inline js/plugins.js]: oh, here should be only global.jQuery

**Comment by drKraken** [comment]
@arthurvr  changed it c354f511a486a0d1ae52c4625835f900b54e97cd

**Pull request merged**